common/src/main/java/com/microsoft/identity/common/internal/logging/Logger.java

```diff
@@ -0,0 +1,210 @@
+package com.microsoft.identity.common.internal.logging;
+
+import android.os.Build;
+import android.util.Log;
+
+import com.microsoft.identity.common.adal.internal.util.StringExtensions;
```

Comment:
there is a string extension class in common core

Reply:
This is the string extension class in common core I think. :)

Reply:
i meant common common, this path is common.adal